packages/host/src/Instance.js

```diff
@@ -371,24 +391,24 @@ rcon.print(game.table_to_json(players))`.replace(/\r?\n/g, " ");
 				type = "special";
 			}
 
-			list.push({
-				name,
+			list.push(new libData.SaveDetails(
```

Comment:
Same here - when passing a structure - the opening `{` on the same line works well. But when we are talking creation of a new object, I feel it's best to have it on its own line
```
list.push(
    new libData.SaveDetails(
        type,
		name,
		stat.size,
		stat.mtimeMs,
		name === loadedSave,
		name === defaultSave,
	)
);
```

Reply:
See line 438 of this file where it's done this way already :)
Maybe there is a style linter/fixer tool that can handle stuff like this? We have CS-Fixer in PHP that handles things like this with breeze.

Reply:
I haven't found any formatters that isn't going to reformat everything to what it thinks is the way to do it and ESLint does not seem to have any rules for this specific case (maybe there's a plugin for that?) This style is used in quite a few places, like the commands in ctl, so changing it I think should be a separate issue/PR.

Reply:
At least you set it up once and then just use it to ensure code style.
You can play with it and see what results it provides and settle on the code-style rules. Auto-formatting helps that it can be run automatically and fix most things without the user even intervening :)